content/en/news/kubecon-china-2021.md

```diff
@@ -0,0 +1,81 @@
+
+- The key components that make a FaaS platform, including function framework, function build, function serving, and function event management.
+- The advantage of the emerging cloud native serverless technologies in each of the key areas of FaaS including Knative Serving, Cloud Native Buildpacks, Shipwright, Tekton, KEDA, and Dapr.
+- How to build a powerful modern FaaS platform with these cloud native technologies by the example of OpenFunction
+- Why does event management matter for FaaS? 
+- Why OpenFunction create its own event management system "OpenFunction Events" when there're already Knative eventing and Argo Events?
+
+## Speaker
+![Huo-Binjie](/images/news/kubecon-china-2021/Huo-binjie.png)
+Benjamin Huo - Founder of OpenFunction
+
+Benjamin Huo led the KubeSphere Observability and Serverless team. He is the creator of FluentBit Operator and the founder of the FaaS project [OpenFunction]((https://github.com/OpenFunction/OpenFunction)), also the author and architect of several observability open source projects such as Kube-Events, Notification Manager, etc. He loves cloud-native and open source technologies and is the contributor of Prometheus.
+
+![Lei-Wanjun](/images/news/kubecon-china-2021/Lei-Wanjun.png)
+Wanjun Lei - KubeSphere Dev Lead, QingCloud Technologies.
+
+Wanjun Lei is the maintainer of OpenFunction and is responsible for developing OpenFunction. He is also the maintainer of FluentBit Operator, and a member of the KubeSphere Observability team, where he is responsible for the development of Notification Manager. He loves cloud native and open source technologies, and is a contributor to fluent bit and nats.
+
+## Session 4: KubeSphere use case sharing in the OpenEBS Office Hours
+
+![openebs-office-hours](/images/news/kubecon-china-2021/Wan-openebs-office-hours.png)
+In cloud-based Kubernetes clusters, persistent storage services are usually provided by cloud providers. When enterprises build an on-premise Kubernetes platform or adopt a Kuberntes distribution in production, persistent storage is the biggest challenge. OpenEBS automates the management of storage attached to the Kubernetes worker nodes and allow the storage to be used for dynamically provisioning OpenEBS PVs or Local PVs. KubeSphere is an open source container platform built on Kubernetes, it integrates OpenEBS as the default persistent storage to provide out-of-the-box persistent storage services for users.
+
+In this talk, Feynman Zhou and Stone Shi from KubeSphere Team will introduce how they leverage OpenEBS and Kubernetes to build a container platform and run stateful workloads on it. 
+
+
+## Session 5: AWS invited KubeSphere Product Manager to speak at the interview
+
+![AWS-Interview](/images/news/kubecon-china-2021/AWS-Interview.png)
+
+
+## RSVP
+
```

Comment:
You can register the KubeCon and CloudNativeCon 2021 via this [link](https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/register/), join us virtually from 9-10 December 2021.